controllers/front/ProductController.php

```diff
@@ -455,7 +455,7 @@ public function displayAjaxRefresh()
             'product_minimal_quantity' => $minimalProductQuantity,
             'product_has_combinations' => !empty($this->combinations),
             'id_product_attribute' => $product['id_product_attribute'],
-            'product_title' => $product['title'],
+            'product_title' => $product['meta_title'],
```

Comment:
```suggestion
            // We use the meta_title key because product_title is used for the title of the page
            'product_title' => $product['meta_title'],
```